rfcs/text/0041/asset.yml

```
@@ -0,0 +1,147 @@
+- name: asset.category
+    level: extend
+    level: extend
+    type: keyword
+    example: hardware
+    description: A further classification of the asset type beyond event.category. Example for host assets {hardware, virtual, container, node}. 
+- name: asset.type
+    level: extend
+    level: extend
+    type: keyword
+    example: workstation
+    description: A sub-classification of assets. For host assets {workstation, S3, Compute}. For user assets {NULL?}
+- name: asset.id
+    level: extend
+    type: keyword
+    example: 2950
+    description: A unique ID for the asset. For inventory integrations, it's the id generated from the inventory data source
+- name: asset.name
+    level: extend
+    type: keyword
+    example: Sourin Paul Macbook Pro
+    description: A common name for the asset
+- name: asset.vendor
+    level: extend
+    type: keyword
+    example: Apple
+    description: Used primarily for 'Host' entities, the vendor name or brand associated with the asset.
+- name: asset.product
+    level: extend
+    type: keyword
+    example: MacBook Pro
+    description: Used primarily for 'Host' entities, the product name associated with the asset.
+- name: asset.model
+    level: extend
+    type: keyword
+    example: TBD
+    description: Used primarily for 'Host' entities, the model name or number associated with this asset.
+- name: asset.version
+    level: extend
+    type: keyword
+    example: TBD
+    description: Used primarily for 'Host' entities, the version or year associated with the asset.
+- name: asset.owner
+    level: extend
+    type: keyword
+    example: sourin.paul@elastic.co
+    description: The primary user entity who owns the 'Host' asset
+- name: asset.priority
+    level: extend
+    type: keyword
+    example: Priority 1
+    description: A priority classification for the asset obtained from outside this system, such as from some external CMDB or Directory service.
+- name: asset.criticality
+    level: extend
+    type: keyword
+    example: Critical
+    description: A criticality classification obtained from outside this system, such as from some external CMDB or Directory service.
+- name: asset.business_unit
+    level: extend
+    type: keyword
+    example: Analyst Experience
+    description: Business Unit associated with the asset (user or host).
+- name: asset.costCenter
```

```suggestion
- name: asset.cost_center
```

nit: use snake over camel casing.